### ISSUE TITLE
Items not placing correctly on Tavern Wall w/ Floor assets

### ISSUE THREAD

**Issue by Mikel-kun** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#434]
**Describe the bug**
Not sure what it is but I think Talespire is registering the little lip on the floor as a 1 tile high object

**To Reproduce**
1. Enter the Tavern Tab through the Building Assets menu
2. Place any Tavern Wall w/ Floor block down (excluding the Tavern Fireplace, Wall 02, Tavern Wall/Floor with Tools, and Tavern Wall/Floor Corner w/ Tools [the second to last one under the Tavern Tab])
3. Place either Bookshelf 01 or 02 on the floor up against the wall.
4. Levitating Magic Bookshelf

**Screenshots/Videos/Twitch Clips**
![720620_screenshots_20200419022824_1](https://user-images.githubusercontent.com/34955611/79681294-08da6d00-81ce-11ea-99e1-6b3cfdf89125.jpg)

**Additional notes**
There's a typo in the "Tavern Fireplace Wall/Floor" asset, it says "Taven' instead of "Tavern"

**Comment by PropForge** [comment]
Is this similar to https://github.com/Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker/issues/273?

**Comment by Mikel-kun** [comment]
Yes, actually, didn't pop up in the "similar issues" thing. Sorry about that!
